Libraries/Components/WebView/WebView.ios.js

```diff
@@ -268,6 +282,13 @@ class WebView extends React.Component {
     ]),
 
     /**
+     * @ios
+     *
+     * Injects a postMessage global into the webview. Required to use onMessage.
+     */
+    messagingEnabled: PropTypes.bool,
```

Comment:
why do we need a `messagingEnabled` prop? What's the issue with always enabling it?